tests/core/test_utils.py

```diff
@@ -32,3 +38,126 @@ def test_profile(self):
             with rasterio.open(outfile) as dataset:
                 self.assertEqual(dataset.compression,
                                  rasterio.enums.Compression.lzw)
+
+    def test_antimeridian_split(self) -> None:
```

Comment:
These tests currently live in "CogifyTest", which is off...can just make another TestCase in this same file specific to these utility methods.

Reply:
Since we're using pytest now I just moved them to the top level: 64986e779309817615519b358e53bc4a3bf6baf7.